### PR TITLE
Add applications/segyio-catb

### PR DESCRIPTION
**Pull request by haavarop**
Segyio-catb is a simple cat-like program that prints binary header of a SEG-Y file to stdout.
Add because it is a feature that has been expressed by the users of SegyIO

### CODE REVIEW

File: applications/CMakeLists.txt [hunk 1]
```diff
@@ -22,7 +22,16 @@ target_compile_definitions(segyio-cath PRIVATE
...
+    -Dsegyio_MAJOR=${segyio_MAJOR}
+    -Dsegyio_MINOR=${segyio_MINOR}
+)
 install(TARGETS segyinfo DESTINATION ${CMAKE_INSTALL_PREFIX}/bin)
 install(TARGETS segyinspect DESTINATION ${CMAKE_INSTALL_PREFIX}/bin)
 
 install(TARGETS segyio-cath DESTINATION ${CMAKE_INSTALL_BINDIR})
+install(TARGETS segyio-catb DESTINATION ${CMAKE_INSTALL_BINDIR})
+
+add_python_test(python.segyb        test/test_segyb.py)
```
Comment (kjellkongsvik) [inline applications/CMakeLists.txt]: Add newline here

File: applications/segyio-catb.c [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+#include <unistd.h>
+
+#include <segyio/segy.h>
+
+static int printhelp(){
+    puts( "Usage: segyio-catb [OPTION]... [FILE]...\n"
+          "Concatenate the binary header from FILE(s) to seismic unix"
+          "output.\n"
+          "\n"
+          "--version    ouput version information and exit\n"
```
Comment (kjellkongsvik) [inline applications/segyio-catb.c]: typo: missing t in output

File: applications/CMakeLists.txt (hunk 1 above)
Comment (haavarop) [inline applications/CMakeLists.txt]: Noticed this was the case in all the files in the PR, so I've added newline to them all

File: applications/segyio-catb.c (hunk 2 above)
Comment (haavarop) [inline applications/segyio-catb.c]: Fixed!

File: applications/segyio-catb.c [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+    for( int i = optind; i < argc; ++i ){
+        segy_file* fp = segy_open( argv[ i ], "r" );
+        if( !fp ) printf("segyio-catb: %s: No such file or directory\n",
+                           argv[ i ] );
+        if( !fp ) continue;
+
+        char binheader[ SEGY_BINARY_HEADER_SIZE ];
+        int err = segy_binheader( fp, binheader );        
+        if( err ) printf( "segyio-catb: %s: Unable to read binary header\n",
```
Comment (jokva) [inline applications/segyio-catb.c]: Wrong channel - error message should go to `stderr`.

File: applications/segyio-catb.c [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+            case 'h': return printhelp();
+            default:  return printhelp();
+        }
+    }
+
+    if( version ) return printversion();
+
+    for( int i = optind; i < argc; ++i ){
+        segy_file* fp = segy_open( argv[ i ], "r" );
+        if( !fp ) printf("segyio-catb: %s: No such file or directory\n",
```
Comment (jokva) [inline applications/segyio-catb.c]: Should go to `stderr`

**Pull request closed without merging**